.obs/specfile/elemental-operator/elemental-operator.spec

```diff
@@ -15,6 +15,8 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+%define commit _replaceme_
```

Comment:
are the "_replaceme_" managed in OBS ? Guess so, but worth double checking... 😄

Reply:
Not really by OBS, they are managed by the scripts in rancher/elemental-obs that pull this code. But yes, I should add a comment here so this is explicit, I think I did that already for the toolkit, should the same here.

Reply:
well, was just a curiosity, I think you can merge as is 👍🏼